client/src/pages/Login.js

```diff
@@ -9,7 +9,7 @@ import {
   loginWithEmail,
   signup,
 } from "../actions/authActions";
-import SubmitButton from "~/components/Button/SubmitButton";
+import CustomButton from "~/components/Button/CustomButton";
```

Comment:
Please use SubmitButton instead. CustomButton was removed.

Reply:
ok

Reply:
I think you also need to pull the latest changes because CustomButton shouldn't be there anymore.

Reply:
i have done that but i encountered an issue https://fightpandemics.slack.com/archives/C010V0DDHLN/p1588782783051600